src/char/char.h

```diff
@@ -142,7 +144,8 @@ struct CharServ_Config {
 	int log_inter;	// loggin inter or not [devil]
 	int char_check_db;	///cheking sql-table at begining ?
 
-	struct point start_point; // Initial position the player will spawn on server
+	struct point start_point[MAX_STARTPOINT]; // Initial position the player will spawn on server
```

Comment:
I'd like to do this in a later commit. It would be nice to refactor some things into vectors.